README.md

```diff
@@ -8,84 +8,9 @@ A simple task manager built with HTML, CSS, and JavaScript using Dragula and Van
 - Move tasks between columns: To Do, Doing, Done, and Trash
 - Clear tasks in the Trash column
 
-<h2 align="center"> This Project Is Perfect For Your First Pull Request </h2>
+## Getting Started
 
-<h1 align="center"> Contributing to this Repo </h1>
+1. Clone the repository:
+   ```bash
+   git clone https://github.com/dormeneur/To-Do-List.git
 
```

Comment:
I had made some changes to the readme file, so could you pull those first to keep them intact?